common/oauthTokenManager.go

```diff
@@ -343,8 +442,23 @@ func (uotm *UserOAuthTokenManager) HasCachedToken() (bool, error) {
 	return uotm.credCache.HasCachedToken()
 }
 
-// RemoveCachedToken delete all the cached token.
+// RemoveCachedToken deletes the token and the AuthenticationRecord as part of the logout command execution.
+// Returns an error if encountered during file deletion or checking.
 func (uotm *UserOAuthTokenManager) RemoveCachedToken() error {
+	filePath := filepath.Join(AzcopyJobPlanFolder, defaultAuthFileName)
```

Comment:
We need to remove the record here and if that fails still we shall try to delete the credCache.

Reply:
RemoveCachedToken() function is invoked only in the case of Logout command execution and this is by default in our original code , the design of this function has been kept such that if we call the logout command :

- we will delete the Authentication record 
-  if we face some problems in deleting report that problem
- in case the Authentication record file does not exist we simply display it over the console
- Complete the original flow of deleting the token in the credcache.